sdk/identity/identity/review/identity.api.md

```diff
@@ -70,9 +70,17 @@ export class ClientSecretCredential implements TokenCredential {
 export class CredentialUnavailable extends Error {
 }
 
+// @public
+export const DEFAULT_AZURE_CREDENTIAL_CHAIN_VERSION = "1";
```

Comment:
can we make "latest" an option too rather than making them assume `DEFAULT_AZURE_CREDENTIAL_CHAIN_VERSION` will be latest? Then we can just default to latest.

Reply:
Thank you, @xirzec